### PR TITLE
Mentee dashboard flushed out

### PR DESCRIPTION
**Pull request by taylorkangillescas**
## Description

Mentee dashboard and navigation created. Components and routing still needs to be implemented.

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

https://trello.com/c/cupZP6nd/43-as-a-mentee-upon-logging-in-id-like-to-have-a-dashboard-with-access-to-a-calender-be-able-to-see-progress-of-courses-and-view-pr

Loom Video: https://www.loom.com/share/9dd21670dd5f4051bfa738f57294cfeb

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by kennymiesner — APPROVED** [review]
Great job getting this copied over from Vaibhavi's branch and implemented. Another example of the puzzle pieces starting to fall into place.

**Review by iwood55 — APPROVED** [review]
It looks great so far and once the calendar is merged to main it will look even better. There are a few lines in the code where things are defined but not used, I know that does not throw any errors but I am curious if they could be taken out or if they are necessary for something in the future.

**Review by tfbielawski — APPROVED** [review]
Nice work. The only thing I noticed was line 4 in MenteeContainer.js, where you have code commented out. Other than that, looks good!

**Review by mariuscmoldovan — APPROVED** [review]
Nice work. A dedicated dashboard for the Mentor/Mentee is nice progress for the project.

**Review by gamboaldo — APPROVED** [review]

File: src/components/pages/Dashboard/MenteeDash/RenderMenteeDash.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+} from '@ant-design/icons';
+
+const { SubMenu } = Menu;
+const { Header, Content, Sider } = Layout;
+
+function onPanelChange(value, mode) {
+  console.log(value.format('YYYY-MM-DD'), mode);
+}
+
+function RenderMenteeDash(props) {
```
Comment (gamboaldo) [inline src/components/pages/Dashboard/MenteeDash/RenderMenteeDash.js:25]: great job on the dashboards

**Review by JorshuaG — APPROVED** [review]
Looks great, these dashboards getting completed are a important step for completing more components and features. Great job

**Pull request merged**